src/aws-cpp-sdk-core/source/Aws.cpp

```diff
@@ -16,6 +16,8 @@
 #include <aws/core/config/AWSProfileConfigLoader.h>
 #include <aws/core/internal/AWSHttpResourceClient.h>
 
+#include <aws/crt/Api.h>
```

Comment:
Should we wrap this in conditional compile path?

Reply:
that's "AWS-CRT-CPP" wrapper, not the http client from CRT.